### PR TITLE
47 contract detail view

### LINKED ISSUE #47: Contract Detail View
## Description
As a purchaser I want to see details about a contract in order to utilize discount.

## Acceptance criteria
- [x] When a contract card is clicked, a details view is shown
- [x] The details view contains (in Swedish):
  - [x] A contract image
  - [x] A contract name
  - [x] How to utilize the discount
  - [x] A description of the discount
  - [x] Contact info for the supplier
  - [x] Text about the supplier
  - [x] Show file download links
  - [x] There is a close button
- [x] An inspirational image is shown along with the contract
- [x] An admin can create clickable links in "How to utilize" text box

## Dependencies
- #30

### PR DESCRIPTION
**Pull request by martinjonsson01**
## Summary
Adds a detail view that is visible when a contract in the contracts page is clicked. 
Also includes a richer editing experience in the admin page, so the admin can include links that are visible in the details view.

chrome_JbWfZy7g9H

## Definition of done
- [x] All acceptance criteria are completed.
- [x] The code has (at least) 80% branch test coverage.
- [x] All tests pass.
- [x] The code is reviewed and accepted by (at least) two other team members.
- [x] Views adhere to design style.

Closes #47

### CODE REVIEW

**Review by JacobBredin — APPROVED** [review]
LGTM! Very reasonable solution 🚢

File: ContractPortal.sln.DotSettings [hunk 1]
```diff
@@ -8,6 +8,7 @@
 	<s:Boolean x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/INT_ALIGN_PROPERTIES/@EntryValue">False</s:Boolean>
 	<s:Boolean x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/INT_ALIGN_PROPERTY_PATTERNS/@EntryValue">False</s:Boolean>
 	<s:Boolean x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/INT_ALIGN_VARIABLES/@EntryValue">False</s:Boolean>
+	<s:String x:Key="/Default/CodeStyle/CodeFormatting/CSharpFormat/WRAP_ARRAY_INITIALIZER_STYLE/@EntryValue">CHOP_IF_LONG</s:String>
```
Comment (jonathan-lindqvist) [inline ContractPortal.sln.DotSettings:11]: 🔪

File: src/Client/Pages/Contracts/ContractDetails.razor [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                        </Body>
+                    </AccordionItem>
+                    <AccordionItem>
+                        <Title>
+                            <h2>Hur man nyttjar erbjudandet</h2>
+                        </Title>
+                        <Body>
+                        <MarkdownViewer Markdown="@Contract.Instructions"/>
+                        </Body>
+                    </AccordionItem>
```
Comment (jonathan-lindqvist) [inline src/Client/Pages/Contracts/ContractDetails.razor:33]: NIT: To minimize excise for the users, is it possible to have one open by default? Though it is hard to guess which one the users want to be open by default.

File: src/Client/Pages/Contracts/ContractsPage.razor [hunk 3]
```diff
@@ -7,9 +7,9 @@
 }
 else
 {
-    @foreach (var contract in _contracts)
+    @foreach (Contract contract in _contracts)
     {
-        <ContractCard Name=@contract.Name ImagePath=@contract.SupplierLogoImagePath/>
+        <ContractCard Contract="contract"/>
```
Comment (jonathan-lindqvist) [inline src/Client/Pages/Contracts/ContractsPage.razor:12]: yes 👍

**Review by jonathan-lindqvist — APPROVED** [review]
Just one NIT 👍

File: src/Client/Pages/Contracts/ContractDetails.razor (hunk 2 above)
Comment (martinjonsson01) [reply src/Client/Pages/Contracts/ContractDetails.razor:33]: Maybe you didn't watch that far in the gif, but the "Om erbjudandet" section is already open by default 😜

**Pull request merged**